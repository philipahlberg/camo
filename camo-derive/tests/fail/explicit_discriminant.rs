use camo_derive::Camo;

#[derive(Camo)]
enum Foo {
    One = 1,
}

fn main() {}
