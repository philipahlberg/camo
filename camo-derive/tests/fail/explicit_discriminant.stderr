error: `camo` does not support explicit discriminants
 --> tests/fail/explicit_discriminant.rs:5:11
  |
5 |     One = 1,
  |           ^
